1-exercises/A-this/exercise-2.js

```diff
@@ -11,10 +11,12 @@ let person = {
     name: "Maggie",
     age: 26,
     currentAddress: "London",
-    changeAddress: /* todo */
+    changeAddress: function(newAddress) {
+        this.currentAddress = newAddress;
+        return this.currentAddress;
```

Comment:
these methods don't always need to `return` something if they are just updating the object properties.